test/e2e/e2e_test.go

```diff
@@ -79,6 +95,69 @@ func TestYunikornEventStream_E2E(t *testing.T) {
 	}, 100*time.Second, 5*time.Second)
 }
 
+func TestYunikornQueueCreation_E2E(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping e2e test in short mode")
+	}
+	ns := "yunikorn"
+	queueName := "tenant" // a random name
```

Comment:
What do you think about generating a small random string here, instead of a fixed-value? That way, there's no chance (or at least, a very small chance) that if this test func is run more than once against the same YK instance, there won't be this queue that was made in a previous run.  Maybe using the first, basic solution in `https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-go` ?

Reply:
I see there's an existing `GenerateRandomAlphanum(t *testing.T, length int) string)` func in YHS - feel free to use that.

Reply:
updated